projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -34,6 +38,11 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
   //   importPath: SPARTACUS_STOREFRONTLIB,
   //   comment: `'${VARIANT_STYLE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
   // },
+  {
+    node: FEATURE_MODULES_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${FEATURE_MODULES_SERVICE}' was removed. Use CmsFeaturesService instead.`,
```

Comment:
nitpick: I would use `CMS_FEATURES_SERVICE` variable here if it's created

Reply:
👍